server/api.go

```diff
@@ -1147,9 +1171,15 @@ func (a *API) dismissNotifications(w http.ResponseWriter, r *http.Request) {
 
 	// At this point, it might be /another/ post by the bot in the DM with that user, but we
 	// allow this for now.
-	err = a.p.apiClient.Post.DeletePost(actionHandler.PostId)
+
+	post.Message = "To change your notification settings, open your user settings or run `/msteams notifications`"
```

Comment:
```suggestion
	post.Message = "You will not receive notifications from Microsoft Teams. To change your notification settings, open your user settings or run `/msteams notifications`"
```

Reply:
Today, `Dismiss` just means that: dismiss the message. If, by chance, you had enabled the setting separately and only then went to dismiss this post, it would not be true that you had chosen not to receive notifications.

We could change `Dismiss` to `Disable` to make this explicit, or we could leave the message ambiguous to represent either starting state, or we could even look up the current setting and add two new messages instead.

Thoughts, @matthewbirtch!

Reply:
@lieut-data I think it makes sense to change `Dismiss` to `Disable` to make it explicit.

Reply:
Sounds good, thank you @matthewbirtch!